tests/test_exchange_universe.py

```diff
@@ -41,6 +46,9 @@ def test_look_up_by_chain_and_slug(persistent_test_client: Client):
     assert quickswap
     assert quickswap.name == "Quickswap"
 
-    # there should be only 1 quickswap on Polygon
-    nonexist = exchange_universe.get_by_chain_and_slug(ChainId.ethereum, "quickswap")
-    assert nonexist is None
+    with pytest.raises(ExchangeNotFoundError) as excinfo:
+        # there should be only 1 quickswap on Polygon
+        nonexist = exchange_universe.get_by_chain_and_slug(ChainId.ethereum, "quickswap")
+        assert nonexist is None
+    
+    assert excinfo.value.args[0] == 'The trading universe does not contain data on chain ethereum for exchange_slug quickswap. This might be a problem in your data loading and filtering. \n                \n    Use tradingstrategy.ai website to explore DEXs.\n    \n    Here is a list of DEXes: https://tradingstrategy.ai/trading-view/exchanges\n    \n    For any further questions join our Discord: https://discord.com/invite/en8tW6MDtw'
```

Comment:
Change the direct Discord link to `https://tradingstrategy.ai/community` as the link may expire